src/server/helpers/achievement.js

```diff
@@ -18,6 +18,8 @@
 
 'use strict';
 
+/* eslint prefer-spread: 1, prefer-reflect: 1, no-magic-numbers: 0 */
```

Comment:
Is there a particular reason these are specified here? Are they intended to be temporary pending style cleanup? If so, a `XXX` comment would be helpful.